ngo_purchase/__init__.py

```diff
@@ -1,2 +1,21 @@
 # -*- coding: utf-8 -*-
+#
+#
+#    Copyright 2015 Camptocamp SA
+#    Author: Alexandre Fayolle
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
```

Comment:
# -*- coding: utf-8 -*-
# See LICENSE file for full copyright and licensing details.